wasm/lib/src/vm_class.rs

```diff
@@ -0,0 +1,78 @@
+use js_sys::TypeError;
+use rustpython_vm::VirtualMachine;
+use std::cell::RefCell;
+use std::collections::HashMap;
+use wasm_bindgen::prelude::*;
+
+// It's fine that it's thread local, since WASM doesn't even have threads yet
+thread_local! {
+    static STORED_VMS: RefCell<HashMap<String, VirtualMachine>> = RefCell::default();
+}
+
+#[wasm_bindgen(js_name = vms)]
```

Comment:
I'd prefer to name it `vmStore` or something that matches the rust code. If we want a shorter name we can have some JavaScript code that alias it.

Reply:
Also it would be nice to have an alias `rustPython` for `rp`, but that's a separate task.

Reply:
In JS code, we could just do `import rustPython from "rustpython_wasm";`; it doesn't require anything on the rust side.